src/backend/api/Fusion.Resources.Api/Controllers/ApiErrors.cs

```diff
@@ -42,12 +42,12 @@ internal static ActionResult InvalidPageSize(string message)
             };
         }
 
-        internal static ActionResult NotFound(string resourcePath)
```

Comment:
The rfc allows for an instance to be defined. Perhaps allow for bot message and optional resource path?